public/wp-content/plugins/ccs-salesforce/includes/cli-commands.php

```diff
@@ -130,6 +131,16 @@ class Import
      */
     protected $supplierSearchClient;
 
+     /**
+     * @var \App\Search\ReindexSearchClient
+     */
+    protected $reindexFrameworkSearchClient;
```

Comment:
Because both the `$reindexFrameworkSearchClient` and `$reindexSupplierClient` inherit the parent class `ReindexSearchClient`. The only difference being paramater in the constructer which either going to be `'frameworks'` or `'supplier'`. This determines what index is going to be reindexed in the context of the ReindexSearchClient class. Does that make sense? If not I'll be happy to eloborate on a call or chat.